lib/Doctrine/DBAL/Platforms/OraclePlatform.php

```diff
@@ -410,6 +428,40 @@ protected function _getCreateTableSQL($table, array $columns, array $options = a
         return $sql;
     }
 
+    /**
+     * Returns the SQL for a list of indexes in the database.
+     *
+     * @param string $database The database schema. If NULL or '/', the currently active schema will be queried.
+     * @param string $table    The table. If left NULL, the SQL will return all the indexes in the database.
+     *
+     * @return string
+     */
+    private function getListIndexesSQL(?string $database, ?string $table): string
+    {
+        $databaseCondition = $this->getDatabaseCondition($database);
+        $tableCondition = '';
+        if ($table !== null) {
+            $tableIdentifier = $this->normalizeIdentifier($table);
+            $quotedTableIdentifier = $this->quoteStringLiteral($tableIdentifier->getName());
+            $tableCondition = " AND ind_col.table_name = $quotedTableIdentifier";
+        }
+        return
+<<<SQL
```

Comment:
Should be on the same line as return, but I'm not sure ATM how this is handled by CS.